runtime/integration-tests/src/cases/lp/investments.rs

```diff
@@ -165,7 +155,7 @@ mod with_pool_currency {
 	use super::{utils, *};
 	use crate::cases::lp::utils as lp_utils;
 
-	#[test_runtimes(all)]
+	#[test_runtimes([development])]
```

Comment:
@wischli I am curious why these are changed only for development; they should be changed to `all`, right?

Reply:
Yeah that was done to simplify CLI during debugging. We can switch back all to `all`!

Reply:
Let's see how time CI will take after that 😆